packages/pdk-nag/src/packs/aws-prototyping.ts

```diff
@@ -0,0 +1,28 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+
+import { CfnResource } from "aws-cdk-lib";
+import { NagPack, NagPackProps } from "cdk-nag";
+import { IConstruct } from "constructs";
+import { PackName, RuleMetadata } from "./aws-prototyping-rules";
+
+/**
+ * Check Best practices for prototypes
+ *
+ */
+export class AwsPrototypingChecks extends NagPack {
+  constructor(props?: NagPackProps) {
+    super(props);
+    this.packName = PackName;
+  }
+  public visit(node: IConstruct): void {
```

Comment:
missing jsdocs

Reply:
Fixed in [43523519d4a565d9d12cb76e4e1335a25fcea7b2](https://github.com/aws/aws-prototyping-sdk/pull/213/commits/43523519d4a565d9d12cb76e4e1335a25fcea7b2)